package.json

```diff
@@ -13,6 +13,7 @@
     "parser": "babel-eslint"
   },
   "dependencies": {
+    "connected-react-router": "^5.0.1",
```

Comment:
Let's remove this and bump to the latest otp-react-redux once https://github.com/opentripplanner/otp-react-redux/pull/66 is merged and released.